library/core/src/main/java/com/google/android/exoplayer2/source/ConcatenatingMediaSource.java

```diff
@@ -297,6 +304,79 @@ public final synchronized void removeMediaSource(
     }
   }
 
+  /**
+   * Removes a range of {@link MediaSource}s from the playlist, by specifying an initial index
+   * (included) and a final index (excluded).
+   *
+   * <p>Note: when specified range is empty, no actual media source is removed and no exception
+   * is thrown.
+   *
+   * @param fromIndex The initial range index, pointing to the first media source that will be
+   *     removed. This index must be in the range of 0 &lt;= index &lt; {@link #getSize()}.
+   * @param toIndex The final range index, pointing to the first media source that will be left
+   *     untouched. The last media source to be removed is at index {@code toIndex} - 1.
```

Comment:
"The last media source to be removed is at index {@code toIndex} - 1."  - That seems redundant and can be removed.